app/views/unattended/provisioning_templates/snippet/puppet_setup.erb

```diff
@@ -17,6 +17,7 @@ os_family = @host.operatingsystem.family
 os_major  = @host.operatingsystem.major.to_i
 os_name   = @host.operatingsystem.name
 
+# EOL Code block - Remove when perforce puppet obsoleted
 aio_enabled = host_param_true?('enable-puppetlabs-repo') || host_param_true?('enable-official-puppet8-repo') || host_param_true?('enable-puppet8') || host_param_true?('enable-official-puppet7-repo') || host_param_true?('enable-puppet7') || host_param_true?('enable-puppetlabs-puppet6-repo') || host_param_true?('enable-puppet6') || host_param_true?('enable-puppetlabs-puppet5-repo') || host_param_true?('enable-puppet5')
```

Comment:
I don't consider it completely obsolete. There are distros (Debian and Fedora) that package it natively and I'd still like to support those.